CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/plugins/io/screenshot/RecentGraphScreenshotUtilities.java

```diff
@@ -166,10 +213,26 @@ public static void refreshScreenshotsDir() {
             filesInDirectory.addAll(Arrays.asList(screenShotsDir.listFiles()));
         }
 
-        RecentFiles.getUniqueRecentFiles().forEach(item -> filesInHistory.add(item.getFileName() + FileExtensionConstants.PNG));
+        RecentFiles.getUniqueRecentFiles().forEach(item -> {
+            filesInHistory.add(item.getFileName() + ".png");
```

Comment:
ditto

Reply:
Sorry I missed this one @antares1470. I also added another one in my most recent commit.